modules/administration-guide/pages/upgrading-the-chectl-management-tool.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: PROCEDURE
+:navtitle: Upgrading the chectl management tool
+:keywords: overview, upgrading, chectl, cli
+:page-aliases:
+
+[id="upgrading-the-{prod-cli}-management-tool_{context}"]
```

Comment:
Does this procedure need a title?

Reply:
Title is in the snippet to be downstream friendly (the procedure doesn't exist downstream)

Reply:
Maybe it's too confusing and it would be better to add a section in downstream explaining that to upgrade the CLI, you need to download the new version in the portal.